storage/include/sparse_merkle/internal_node.h

```diff
@@ -217,14 +225,14 @@ class BatchedInternalNode {
   BatchedInternalNode() = default;
 
   // Construct from the passed children container.
-  BatchedInternalNode(const ChildrenContainer& children) : children_{children} {}
+  BatchedInternalNode(const Children& children) : children_{children} {}
 
   // Insert a LeafChild into this internal node. Return a type indicating success or
   // failure that contains the necessary data for the caller (the sparse merkle
   // tree implementation) to do the right thing.
   //
   // `depth` represents how many nibbles down the sparse merkle tree this BatchedInternalNode is.
-  InsertResult insert(const LeafChild& child, size_t depth);
+  InsertResult insert(const LeafChild& child, size_t depth, Version current_version);
```

Comment:
Could you please document the `current_version` parameter as opposed to `child.key.version()`?

Reply:
I saw it in `insertIntoEmptyChild()`.

Reply:
Good idea! Done.